### PR TITLE
allow ingress without host

### PR DESCRIPTION
**Pull request by acristu**
There are cases where explicitly setting the host is not desired, multiplexing to different services can be done via `path`.
The console can be deployed with a `basePath` and ingress without an explicit host,

### CODE REVIEW

**Review by alejandroEsc — COMMENTED** [review]
Can you please update the chart.yaml file to the next patch version. Thank you!

**Comment by joejulian** [comment]
Also, please rebase. I think that'll fix CI.

File: charts/console/templates/ingress.yaml [hunk 1]
```diff
@@ -38,7 +38,9 @@ spec:
   {{- end }}
   rules:
     {{- range .Values.ingress.hosts }}
-    - host: {{ tpl (.host) $ | quote }}
+    - {{- if .host }}
+      host: {{ .host }}
+      {{- end}}
```
Comment (joejulian) [inline charts/console/templates/ingress.yaml:43]: This was previously allowing .host to be a template. This changes removes this feature.
```suggestion
    {{- $host := tpl .host $ }}
    - {{- if $host }}
      host: {{ $host }}
      {{- end}}
```

Comment (acristu) [reply charts/console/templates/ingress.yaml]: thanks for the review, I've re-added the `tpl` , I can add the `$host` also if you want, but for now I just put the previous code back

**Review by RafalKorepta — APPROVED** [review]

**Comment by joejulian** [comment]
If you'll check that allow changes box, we can bump the chart version for you as needed to get this merged. For now, it needs rebased and the chart version needs bumped again.

File: charts/console/templates/ingress.yaml (hunk 1 above)
Comment (joejulian) [reply charts/console/templates/ingress.yaml]: I was thinking that if `tpl .host $` produces an empty string, you may still want it to be skipped. That's why I suggested the `$host` idea.

**Comment by joejulian** [comment]
closing as abandoned. Work moved to #537

**Pull request closed without merging**